### ISSUE TITLE
MudExDialog Position and Size properties

### ISSUE THREAD

**Issue by RDUser-dev** [fgilde/MudBlazor.Extensions#81]
### Feature request type

Enhance component

### Component name

MudExDialog

### Is your feature request related to a problem?

In many circumstances it is necessary to control the position and size of the dialog from the outside for example:
if you want to make the position and size of the window persistent to be able to restore them when the page is refreshed.
It is therefore necessary for the component to expose these properties and the related change event.

### Describe the solution you'd like

The optimal solution would be for the component to expose the position and size properties and the drag and resize events, for example:
```
<MudExDialog
    @bind-Visible="@PopupVisible"
    PositionX="positionX"
    PositionY="positionY"
    DragCompleted="OnDialogDragCompleted"
    Height="height"
    Width="width"
    ResizeCompleted="OnDialogResizeCompleted">
</MudExDialog>

@code{
    int? positionX, positionY, Height, Width;
    protected override async Task OnAfterRenderAsync(bool firstRender)
    {
        if (firstRender)
        {
            var position = await LoadPositionFromLocalStorageAsync();
            (positionX, positionY) = (position?.X ?? null, position?.Y ?? null);

            var size = await LoadSizeFromLocalStorageAsync();
            (height, width) = (size ?.Height ?? null, size ?.Width ?? null);

            StateHasChanged();
        }
    }
   async Task OnDialogDragCompleted(DialogDragCompletedEventArgs args)
    {
        await SavePositionToLocalStorageAsync(args.X,args.Y);
    }
   async Task OnDialogResizeCompleted(DialogResizeCompletedEventArgs args)
    {
        await SaveSizeToLocalStorageAsync(args.Height,args.Width);
    }
}
```

### Have you seen this feature anywhere else?

https://demos.devexpress.com/blazor/Popup#Dragging

### Describe alternatives you've considered

An alternativa can be some bindable properties, for example:
```
<MudExDialog
    @bind-Visible="@PopupVisible"
    Position="position"
    PositionChanged="OnDialogPositionChanged"
    Size="size"
    SizeChanged="OnDialogSizeChanged">
</MudExDialog>
```
But in this case Position and Size must be immutable objects.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fgilde** [comment]
Feature will be released in 1.7.88 not complete finished yet
but for testing its avaliable now in this preview packege https://www.nuget.org/packages/MudBlazor.Extensions/1.7.88-prev-2405261413-main

A using demo can be found  here https://mudex.azurewebsites.net/dialogs-pos-size

If you are using MudExDialog as inline dialog you will have eventcallbacks as mentioned.
```
 [Parameter] public EventCallback<DialogDragStartEvent> OnDragStart { get; set; }
 [Parameter] public EventCallback<DialogDraggingEvent> OnDragging { get; set; }
 [Parameter] public EventCallback<DialogDragEndEvent> OnDragEnd { get; set; }
 [Parameter] public EventCallback<DialogResizedEvent> OnDialogResized { get; set; }
 [Parameter] public EventCallback<DialogResizingEvent> OnDialogResize { get; set; }
 ```
 
 If youre are using non inline dialogs you need to inject the `IDialogEventService`
 
 ```
 [Inject] private IDialogEventService EventService { get; set; }
 ```
 
 then you can subscribe events like
 
 ```
  EventService.Subscribe<DialogDragEndEvent>(MyHandler);    
 ```
 
 But then please ensure unsubscribe in your page dispose with
 
 ```
   EventService.UnSubscribe<DialogDragEndEvent>(MyHandler);    
 ```

**Comment by RDUser-dev** [comment]
Integrated and tested in my application.
Everything works perfectly
Thank you
